code/go/0chain.net/chaincore/chain/block_fetcher.go

```diff
@@ -345,6 +345,7 @@ type FetchedNotarizedBlockHandler interface {
 //go:generate mockery --inpackage --testonly --name=Chainer --case=underscore
 // The Chainer represents Chain.
 type Chainer interface {
+	GetChainConfig() config.ChainConfig
```

Comment:
@peterlimg I'm getting this error: "import cycle not allowed in test".
This is probably because of `--inpackage` flag for `Chainer`?
What would you recommend?

Reply:
Maybe we should move interfaces `conifg.ChainConfig` and `util.MerklePatriciaTrieI` to a separate package(s), like util_interfaces and config_interfaces? or just a single package with all interfaces?

Reply:
@rrrooommmaaa we will need to get the config from MPT from latest finalized block (not loaded yet), before that we can get the chain config from `config.Configuration().ChainConfig` if there are import cycle issue.

Reply:
This `config.Configuration().ChainConfig` is a global variable, and I'm trying to modularize this method

Reply:
Yes, but let's do the modularize work in another PR, let's focus on fixing the issue #1387.